engine/dlib/src/dmsdk/dlib/log.h

```diff
@@ -120,23 +117,28 @@ namespace dmLog
  * Log severity
  *
  * @enum
- * @name dmLog::Severity
- * @member dmLog::LOG_SEVERITY_DEBUG
- * @member dmLog::LOG_SEVERITY_USER_DEBUG
- * @member dmLog::LOG_SEVERITY_INFO
- * @member dmLog::LOG_SEVERITY_WARNING
- * @member dmLog::LOG_SEVERITY_ERROR
- * @member dmLog::LOG_SEVERITY_FATAL
+ * @name LogSeverity
```

Comment:
I wanted to use our logging system from C.
So I needed to put these back into "global scope".

Reply:
ah, ok, makes sense

Reply:
makes sense